src/config/partners/showtime.ts

```diff
@@ -35,7 +35,7 @@ Wrap It Up with Incredible Raffles: Stick around to the end, and you could score
     {
       image: '/partners/showtime/drops/drop-2.jpg',
       creator: '0x7F255F64e81E540144aC41f32fcc25fA33586FFD',
-      name: `BLOND:ISH: BLOND:ISH'S TREASURE CHEST`,
+      name: `BLONDISH's Treasure Chest`,
```

Comment:
Will need to confirm with them, but I'm relatively sure this should be `BLOND:ISH's Treasure's Chest`

Reply:
@aneri18